pkg/jx/cmd/create_jenkins_token.go

```diff
@@ -96,8 +90,8 @@ func NewCmdCreateJenkinsUser(f Factory, in terminal.FileReader, out terminal.Fil
 	options.ServerFlags.addGitServerFlags(cmd)
 	cmd.Flags().StringVarP(&options.ApiToken, "api-token", "t", "", "The API Token for the user")
 	cmd.Flags().StringVarP(&options.Password, "password", "p", "", "The User password to try automatically create a new API Token")
-	cmd.Flags().StringVarP(&options.Timeout, "timeout", "", "", "The timeout if using browser automation to generate the API token (by passing username and password)")
-	cmd.Flags().BoolVarP(&options.UseBrowser, "browser", "", false, "Use a Chrome browser to automatically find the API token if the user and password are known")
+	cmd.Flags().StringVarP(&options.Timeout, "timeout", "", "", "The timeout if using REST to generate the API token (by passing username and password)")
+	cmd.Flags().BoolVarP(&options.UseBrowser, "browser", "", false, "Use REST calls to automatically find the API token if the user and password are known")
```

Comment:
Note that the naming of the option is now misleading, but I was not sure how to deprecate the old option nicely.

(I am not exactly sure why this option needs to be passed at all—I do not see a clear use case for a command that just waits for you to get a token by hand—nor I am clear on why `--password` needs to be specified when `~/.jx/jenkinsAuth.yaml` already stores the password. But these behaviors are orthogonal to my change.)